bedrock/mozorg/views.py

```diff
@@ -493,20 +480,11 @@ def home_tweets(locale):
     return []
 
 
-def new_home(request, locale=None):
+def home(request, locale=None):
```

Comment:
We can now also drop the `locale` param from this function signature. Just set it to `locale = l10n_utils.get_locale(request)` below.